byterun/caml/spacetime.h

```diff
@@ -131,12 +133,17 @@ typedef struct {
 } allocation_point;
 
 typedef struct {
+  value callee_node;
+  value call_count;
```

Comment:
Guarding this is going to cause trouble in the decoding code, I think, which is supposed to cope whether or not the profile contains call counts.  Luckily though the size of `allocation_point` is already larger than the size of `call_point` and they are both in a union together, so this does not increase memory consumption.  So I don't think anything needs changing here.